frontend/mr-admin-flate/src/pages/tiltaksgjennomforinger/TiltaksgjennomforingKnapperad.tsx

```diff
@@ -23,6 +26,8 @@ export function TiltaksgjennomforingKnapperad({ style, tiltaksgjennomforing }: P
         Tilgjengelig for veileder
       </Switch>
 
+      {historikk && <ViewEndringshistorikk historikk={historikk} />}
```

Comment:
Kan vi bruke ternary istedenfor `&&`, ref: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx